### PR TITLE
OOP School Library #4: UI

### PR DESCRIPTION
**Pull request by miltonHenschel**
- [x] Create app file with the methods below:

  - List all books
  - List all persons
  - List all rentals
  - Create a person (teacher or student)
  - Create a book
  - Create a rental

- [x] Create main file with the methods below:

  - Create menu
  - Execute user's choice
  - Exit app

### CODE REVIEW

File: rental.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class Rental
   attr_accessor :date, :person, :book
 
-  def initialize(date, person, book)
+  def initialize(person, book, date = Time.now)
```
Comment (Whoistolu) [inline rental.rb:4]: - [ ] Great job using `Time.now` here instead of Users adding in the date manually 👍🏽

**Review by Whoistolu — APPROVED** [review]
### Hi @miltonHenschel 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

### Highlight
🔆 The school library console app behaves in the same way as the [template design provided 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: rental.rb (hunk 1 above)
Comment (miltonHenschel) [reply rental.rb:4]: Thanks very much @Whoistolu. 😁

**Review by amejid — COMMENTED** [review]
Hi @miltonHenschel, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Correct workflow :heavy_check_mark: 
- Console App working well :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**